pom.xml

```diff
@@ -30,8 +30,35 @@
 
     <properties>
         <shadeBase>org.weakref.jmx.\$internal</shadeBase>
+        <dep.plugin.njord.version>0.8.7</dep.plugin.njord.version>
+        <njord.publishingType>automatic</njord.publishingType>
+        <njord.waitForStates>true</njord.waitForStates>
```

Comment:
Can these go in the plugin configuration, instead?

Reply:
It can. Moved